packages/patrol_cli/lib/src/android/android_test_backend.dart

```diff
@@ -92,6 +100,33 @@ class AndroidTestBackend {
     });
   }
 
+  Future<void> loadJavaPathFromFlutterDoctor() async {
+    final javaCompleterPath = Completer<String?>();
+
+    await _disposeScope.run((scope) async {
+      final process = await _processManager.start(
+        ['flutter', 'doctor', '--verbose'],
```

Comment:
I'm wondering if we are okey with using plain `flutter` here. What will happen in case someone has only flutter from `fvm` or `puro` configured? (Is it even possible?) Maybe we should use `FlutterCommand` instead?

Reply:
Updated